src/uwsgi.ini

```diff
@@ -0,0 +1,13 @@
+[uwsgi]
+strict = true
+master = true
+enable-threads = true
+vacuum = true
+
+processes = 4
+threads = 2
+
+http = :8001
```

Comment:
Het is belangrijk dat ook de ansible playbook aangepast wordt in Openstack! Is daar al een (WIP) PR voor?

In die change moeten de WSGI_* envs weg gehaald worden.

Reply:
check, was ik me van bewust idd. Is er nog niet maar doe ik zo

Reply:
PR aangemaakt